esm/interface/node.js

```diff
@@ -97,8 +94,14 @@ export class Node extends EventTarget {
   get textContent() { return null; }
   set textContent(value) {}
   normalize() {}
-  cloneNode() { return null; }
-  contains() { return false; }
+  /**
+   * @param {boolean | undefined} deep
+   */
+  cloneNode(deep) { return null; }
```

Comment:
`Document`, `Element`, and `NonElementParentNode` take a `deep` parameter

Reply:
they do, this is an interface like implementation

Reply:
or you mean this should be the TSDoc change to make it happy? In such case, it's OK